src/cms/models/requests/request.py

```diff
@@ -0,0 +1,95 @@
+"""Database model for requests
+
+"""
+from django.core.validators import RegexValidator
+from django.db import models
+from cms.models.regions.region import Region
+from cms.models.vehicles.vehicle import Vehicle
+
+
+# pylint: disable=too-few-public-methods
+class RequestManager(models.Manager):
+    def get_queryset(self):
+        # only return true Requests
+        return super(RequestManager, self).get_queryset()
+
```

Comment:
```suggestion
```
I think you can just remove this class, it doesn't seem to do anything different than the default model manager.

Reply:
Have now made this change.